### ISSUE TITLE
Email notification when a part is changed

### ISSUE THREAD

**Issue by SchrodingersGat** [inventree/InvenTree#7834]
### Discussed in https://github.com/inventree/InvenTree/discussions/7819

<div type='discussions-op-text'>

<sup>Originally posted by **jesseyu1984918** August  7, 2024</sup>
I see it's possible to subsribe to a part, but whether it's possible to receive an email when a part is changed?</div>

**Comment by jesseyu1984918** [comment]
Would really be helpful if this feature can be added

**Comment by matmair** [comment]
@SchrodingersGat I would see this more as a plugin using the events plugin

**Comment by SchrodingersGat** [comment]
Agreed. Would be a simple plugin to add in. @jesseyu1984918 would you be interested in implementing this, or supporting via funding a plugin?
